CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 0.6.0 (August 19, 2022)
+
+FEATURES:
+
+* **New Resource:** `steampipecloud_workspace_snapshot`
+
+ENHANCEMENTS:
+
+* `resources/steampipecloud_organization_member`: Remove redundant call to get orgMember. 
+
+BREAKING CHANGES:
```

Comment:
Can you please make this the first section for this version (as they're the most important)?

Reply:
`BREAKING CHANGES` should just move to the top for this version. FYI I'm using https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md as a reference/standard.